PHP/hw3.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+function judgePrime($n){
+    for ($i=2; $i<=$n; $i++) if ($n%$i==0) break;
+    
+    if ($n==1 or $n==$i) echo "$n is primenumber. <br />";
+    else echo "$n is not primenumber. <br />";
```

Comment:
Please try to follow coding standard at https://www.php-fig.org/psr/psr-2/
Let take a look on if-else and for.
Following coding standard helps your source code more clear.